dist/mobile/chico.js

```diff
@@ -7305,6 +7305,302 @@ ch.factory = function (Klass, fn) {
+
+        if (!this._moved) { // check if exists another animation executed
+
+            var that = this;
+
+            // set flag with true value, to prevent multiple animations
+            this._moved = true;
+            
+            // call rAF with carousel animation method how params
+            rAF(function() {
+                that._animateCarousel(x);
+            });
+
+        }
+
+    };
+
+    /**
+     * this._handleStart()
+     * @memberof! ch.Carousel.prototype
+     * @private
+     * @type Function()
+     * @param {Number} touchProperties all properies of currentTouch event
+     */
+    Carousel.prototype._handleStart = function(touchProperties) {
+
+        // Delete efects on list to make changes instantly
+        this._$list.addClass('ch-carousel-nofx');
+
+        // Obtenemos la posición referente al eje X del primer toque
+        this._startTouchX = touchProperties.clientX;
+        this._startTouchY = touchProperties.clientY;
+        this._partialTouchX = touchProperties.clientX;
+
+    };
+
+    /**
+     * this._handleEnd()
+     * Is evaluated based on the movement if done you have to go to the next or previous page of the carousel
+     * @memberof! ch.Carousel.prototype
+     * @private
+     * @type Function()
+     * @param {Number} touchProperties all properies of currentTouch event
+     */
+    Carousel.prototype._handleEnd = function(touchProperties) {
+
+        // get the X position of last touch
+        this._endTouchX = touchProperties.clientX;
+
+        // calculate the total width of touch event
+        var touchWidth = this._startTouchX - this._endTouchX,
+            // force touchWidth to positive value (distance always is positive)
+            touchDistance = Math.abs(touchWidth); 
+        
+        // Activate effects
+        this._$list.removeClass('ch-carousel-nofx');
+
+        if ( touchDistance >= this._minimumTouch ) {  // if the touch distance is higher that minimum touch width
+
+            if (touchWidth > 0) { // if touch go from right to left
+
+                if (this._currentPage === this._pages) { // if last page, don't get de next page
+                    this._captureMove(-this._pageWidth * (this._currentPage - 1));
+                } else {
+                    this.next();
+                }
+
+            } else { // if touch go from left to right
+
+                if (this._currentPage === 1) { // if fist page, don't get de previous page
+                    this._captureMove(0);
+                } else {
+                    this.prev();
+                }
+
+            }
+
+        } else { // if the touch distance is minnor that minimum touch width, 
+            
+            // back carousel to previus position
+            this._captureMove(-this._pageWidth * (this._currentPage - 1));
+
+        }
+
+        // After touch end, available scroll again
+        if (this._disabledScrollY) {
+            document.body.style.overflow = 'initial';
```

Comment:
The `initial` property has a [decent support](http://caniuse.com/#search=initial), but maybe we could use `visible` that's the [default (initial) value](https://developer.mozilla.org/en/docs/Web/CSS/overflow). Thoughts?